documentation/how-to-use.md

```diff
@@ -0,0 +1,78 @@
+In this document, we will introduce how to use JS SDK for very new beginner to typescript SDK.
+
+1. Prepare your environment.  
+    NodeJS: can be installed from https://nodejs.org/en/download/  
+    typescript: install it with `npm install -g typescript`.
+
+1. create a empty folder and cd this folder.
```

Comment:
step 1 repeated?

Reply:
Actually, this is markdown file format, it will show up as incremental number if you view it like this https://github.com/Azure/azure-sdk-for-js/blob/4a6b738cfa68b4c4b0b54942f1560ed8f9c8a755/documentation/how-to-use.md